platform/android/export/export.cpp

```diff
@@ -778,6 +778,75 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		}
 	}
 
+	void _write_tmp_manifest(const Ref<EditorExportPreset> &p_preset, bool p_give_internet, bool p_debug) {
+		String manifest_text =
+				"<?xml version=\"1.0\" encoding=\"utf-8\"?>\n"
+				"<manifest xmlns:android=\"http://schemas.android.com/apk/res/android\"\n"
+				"    xmlns:tools=\"http://schemas.android.com/tools\">\n";
+
+		manifest_text += _get_screen_sizes_tag(p_preset);
+		manifest_text += _get_gles_tag();
+
+		Vector<String> perms;
+		_get_permissions(p_preset, p_give_internet, perms);
+		for (int i = 0; i < perms.size(); i++) {
+			manifest_text +=
+					"    <uses-permission android:name=\"" + perms.get(i) + "\"/>\n";
+		}
+
+		String xr_metadata_text = "";
+		String fa_metadata_text = "";
+		int xr_mode_index = p_preset->get("xr_features/xr_mode");
+		if (xr_mode_index == 1 /* XRMode.OVR */) {
+			String focus_awareness = bool_to_string(p_preset->get("xr_features/focus_awareness"));
+			xr_metadata_text = "<tools:node=\"replace\" meta-data android:name=\"com.samsung.android.vr.application.mode\" android:value=\"vr_only\" />\n";
```

Comment:
`manifest_text +=` ?

Reply:
The metadata tags aren't supposed to be in the manifest in that order.